content/posts/2017/09/2017-09-13-datbriefing-how-blockchain-s-future-growth-will-affect-federal-agencies.md

```diff
@@ -0,0 +1,53 @@
+---
+slug: the-data-briefing--how-blockchain-s-future-growth-will-affect-federal-agencies
```

Comment:
Should match the slug on DG: `the-data-briefing-how-blockchains-future-growth-will-affect-federal-agencies`